process/block/displayBlock.go

```diff
@@ -66,8 +66,8 @@ func (txc *transactionCounter) getNumTxsFromPool(shardId uint32, dataPool dataRe
 	return sumTxs
 }
 
-// substractRestoredTxs updated the total processed txs in case of restore
-func (txc *transactionCounter) substractRestoredTxs(txsNr int) {
+// subtractRestoredTxs updated the total processed txs in case of restore
```

Comment:
Please add a comment to an exported uncomment method in this file -> DisplayLastNotarized

Reply:
Done.